pixart.js

```diff
@@ -1 +1,36 @@
 
+function setBrushBoxColor(event) {
+    brush.style.backgroundColor = colorField.value;
+    event.preventDefault();
```

Comment:
super nit-picky.. but i believe convention would say put the preventDefault first here